app/controllers/api/requests_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  class RequestsController < BaseController
+    def find_requests(id)
```

Comment:
@abellotti these two m ethods are also crying out for a more OO solution. To follow!